Makefile

```diff
@@ -99,7 +99,6 @@ test: unit_test integration_test
 
 define run_unit_test
 	@echo "running unit test for packages:" $(1)
-	bash -x ./tests/wait_for_mysql.sh
```

Comment:
we can not delete this for now

Reply:
you mean CI has ensured that MySQL is started?

Reply:
yes in [here](https://github.com/PingCAP-QE/ci/blob/0168669f271712bcde92dff2603fcf962d90c69b/jenkins/pipelines/ci/dm/dm_ghpr_new_test.groovy#L180)